### PR TITLE
Disguiser: Rework

### PR DESCRIPTION
**Pull request by TimGoll**
Cleaned up the disguiser code and moved the icon to the status system. This way it is only displayed while the player is disguised to remove the redudancy with the text displayed ob top of the HUD.

### CODE REVIEW

File: gamemodes/terrortown/gamemode/shared/lang/italian.lua [hunk 1]
```diff
@@ -639,7 +639,7 @@ L.karma_min = "Irresponsabile"
 L.corpse = "Cadavere"
 L.corpse_hint = "Premi {usekey} per identificare. {walkkey} + {usekey} per identificare segretamente."
 
-L.target_disg = "(TRAVESTITO)"
+L.target_disg = "(travestito)"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/italian.lua:642]: Those brackets shouldn't be a part of the translation

Comment (TimGoll) [reply gamemodes/terrortown/gamemode/shared/lang/italian.lua:642]: They were before and I though it was cheaper that adding two string concats each frame

Comment (Alf21) [reply gamemodes/terrortown/gamemode/shared/lang/italian.lua:642]: Hmm for sure, that's true, but anyways that's not a part of a translation by default...

Comment (TimGoll) [reply gamemodes/terrortown/gamemode/shared/lang/italian.lua:642]: I can add the brackets in the rendering hook if you like that more

File: gamemodes/terrortown/gamemode/client/cl_keys.lua [hunk 2]
```diff
@@ -183,11 +183,5 @@ end
 -- @ref https://wiki.facepunch.com/gmod/GM:PlayerButtonUp
 -- @local
 function GM:PlayerButtonUp(ply, btn)
```
Comment (lebroomer) [inline gamemodes/terrortown/gamemode/client/cl_keys.lua:185]: Why do we need the remaining Hook Stub here?

Comment (TimGoll) [reply gamemodes/terrortown/gamemode/client/cl_keys.lua:185]: maybe for documentation? I dunno

Comment (lebroomer) [reply gamemodes/terrortown/gamemode/client/cl_keys.lua:185]: But this is a default hook, we don't need it in our code - https://wiki.facepunch.com/gmod/GM:PlayerButtonUp

File: gamemodes/terrortown/gamemode/shared/lang/german.lua [hunk 3]
```diff
@@ -1287,9 +1283,12 @@ L.hud_revival_time = "{time}s"
...
---2020-05-28
+-- 2020-05-28
 L.confirm_detective_only = "Nur Detektive können Leichen bestätigen"
 L.inspect_detective_only = "Nur Detektive können Leichen untersuchen"
 L.corpse_hint_no_inspect = "Nur ein Detektiv kann diesen Körper untersuchen."
 L.corpse_hint_inspect_only = "Drücke [{usekey}] zum Durchsuchen. Nur Detektive können diesen Körper bestätigen."
 L.corpse_hint_inspect_only_credits = "Drücke [{usekey}] zum Erhalten der Credits. Nur ein Detektiv kann diesen Körper untersuchen."
+
+-- 2020-06-04
+L.label_bind_disguiser = "Schalte Tarnung"
```
Comment (saibotk) [inline gamemodes/terrortown/gamemode/shared/lang/german.lua:1294]: "Tarnung umschalten"

**Pull request merged**